monitoring/mock_uss/msgsigning/__init__.py

```diff
@@ -0,0 +1,6 @@
+from monitoring.mock_uss import config, SERVICE_MESSAGESIGNING
+
+if config.Config.CERT_BASE_PATH is None:
+    raise ValueError(
+        f"CERT_BASE_PATH is required for the {SERVICE_MESSAGESIGNING} service"
+    )
```

Comment:
Since we're giving a default value to Config.CERT_BASE_PATH, I don't think this can ever be None.  We could check for non-emptiness:

```suggestion
if not config.Config.CERT_BASE_PATH:
    raise ValueError(
        f"Environment variable {config.ENV_KEY_CERT_BASE_PATH} may not be blank for the {SERVICE_MESSAGESIGNING} service"
    )
```